app/src/main/java/com/yacgroup/yacguide/utils/MarkdownUtils.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2020 Christian Sommer
+ * Copyright (C) 2020 2022 Christian Sommer
```

Comment:
Is it best practice to collect all years, the file has changed? I would simply go for the most recent year.
Sth. like `2020, 2022, 2023, 2025` looks strange.

Reply:
I'm not an expert but I think you need to list all years or a range: https://www.gnu.org/licenses/gpl-howto.en.html